### PR TITLE
Da/nuke

### PR DESCRIPTION
**Pull request by tadad**
**Problem**:
way too much code

**Solution**:
delete it all

**Notes**:
some symlinks had to change. Also might have deleted a little too much - we'll find out!

### CODE REVIEW

File: lib/docket.hoon [hunk 1]
```diff
@@ -1 +1 @@
-../../garden-dev/lib/docket.hoon
\ No newline at end of file
+../../webterm/lib/docket.hoon
```
Comment (tadad) [inline lib/docket.hoon]: what desk should we symlink to? Does it matter?

[the remark this answers is not in the document]
Comment (hosted-fornet) [reply lib/docket.hoon]: Seems ok. arvo and base-dev def main preferred; I think tloncorp/desk-dev is more canonical than webterm, but it is also a whole other repo we'll have to tell devs to get, so I think it is ok to use webterm instead.

**Review by hosted-fornet — APPROVED** [review]
Thanks for the cleanup :+1:

**Comment by hosted-fornet** [comment]
Feel free to merge whenever once you've resolved the conflict (I think it is just bringing in some changes from @0x70b1a5 s blit-JSONifying PR).

**Pull request merged**